src/Workspaces/Core/MSBuild/MSBuild/MSBuildProjectLoader.Worker.cs

```diff
@@ -132,7 +137,7 @@ public async Task<ImmutableArray<ProjectInfo>> LoadAsync(CancellationToken cance
                 var results = ImmutableArray.CreateBuilder<ProjectInfo>();
                 var processedPaths = new HashSet<string>(PathUtilities.Comparer);
 
-                _buildManager.StartBatchBuild(_globalProperties);
+                _buildManager.StartBatchBuild(_globalProperties, _msbuildLogger);
```

Comment:
Should the log just be held by the _buildManager rather than having to thread stuff through?